### ISSUE TITLE
Use zwave device titles for filters

### ISSUE THREAD

**Issue by dannybloe** [domoticz/domoticz#4817]
This is very handy already:

![filters](https://user-images.githubusercontent.com/2369994/117610553-3d66f100-b162-11eb-986b-38be8344e783.png)

But it would be a lot beter if it shows the actual device titles as shown in the hardware panel for zwave instead of `Z-Stick xxx (yyy)` (or whatever your hardware is called).

Cheers,

Danny

**Comment by waltervl** [comment]
I guess moving to ZwaveJS2MQTT could make this request obsolete?

**Comment by roblomq** [comment]
In ZwaveJS2MQTT it's even worse because there is no filter possibility at all..

**Comment by gizmocuz** [comment]
> In ZwaveJS2MQTT it's even worse because there is no filter possibility at all..

Why is this? I select the MQTT-AD hardware and see all devices

**Comment by roblomq** [comment]
> > In ZwaveJS2MQTT it's even worse because there is no filter possibility at all..
> 
> Why is this? I select the MQTT-AD hardware and see all devices

It sounded more negative than intended. What I meant was that when I used OZW there where sub-filters, as shown above, that made it able to filter per node. Now when I use ZwaveJS2mqtt there are no sub-filters to filter per node, the only option is to filter on the hardware (MQTT) and then indeed you get all devices. But then I get a list of about 200 devices. And yes I then can use the filter search field so it's solvable...

**Comment by gizmocuz** [comment]
Yes correct. I just enter (part of) the ID and see all related devices. Or just use (part of) the name
I actually never used the filter / node for zwave on the left.
Reason is that I have many devices, and you would have to know the Node ID (Number)

There are also other ways to find your node:
- Group the devices in a room plan that you can select from the dropdown
- Use the (new) search box on each page
- And after a while you know where you dragged/dropped the device on which tab

For me we can close this

And I guess when you named the ZStick -> Aeotec ZStick then it would appear at the top so you could not use the checkboxes for the other hardware unless you scroll down

**Comment by waltervl** [comment]
Not relevant anymore as Openzwave is being in the process to be deprecated.
